portfolio/src/main/webapp/js/script.js

```diff
@@ -53,9 +53,39 @@ async function getText() {
   }
 }
 
+/**
+ * Display text from data Servlet
+ */
+async function getText() {
+  var numComments = document.getElementById("number-comments");
```

Comment:
`var` -> `const`

https://google.github.io/styleguide/jsguide.html#features-use-const-and-let